projects/farmworker-wallet-os.md

```diff
@@ -0,0 +1,128 @@
+
+ 
+
+## Origin & History 
+
+The origins of the **Farmworker Wallet OS** project align with those of Entidad. What began as a volunteering effort by three college friends, was formalized in 2018 with the founding of Entidad as a public benefit corporation.  We have primarily been working with leading farm worker-serving organizations to develop technology that leverages growing digital literacy in their communities to scale their impact.  
+
+ 
+The first digital service launched supported the UFW Foundation’s emergency relief efforts during the height of the COVID-19 pandemic. The solution was used to plan, manage and operate over 500 in-person community events where over $15 million in resources were distributed to over 40K families. Additionally, it has allowed us to better understand the unique challenges of building for underserved communities and why interoperable, secure, and privacy preserving technologies are key to addressing these challenges.  
+
+  
+You can read more about our journey on [our blog](https://www.entidad.io/blog). 
+
+# Alignment with the OpenWallet Foundation Mission
+The **Farmworker Wallet OS** project is a community of contribution led by Entidad and the United Farm Workers Foundation (UFWF) with the goal of furthering the adoption of an open, secure, interoperable [digital wallet solution](https://youtu.be/Uo7CTtGrOw8) that makes it easier for farm worker communities to access life-changing social and human services. The open-source software components maintained by this community will enable non-profit and social good organizations to build wallet-driven solutions on top of the Mendix low-code platform. Collaboration with app makers from the broader Mendix developer community will help inform and guide the Farmworker Wallet OS project with industry proven best practices, leading to the implementation of reusable functionality that is aligned with the goals of the Open Wallet Foundation (to preserve user choice, security and privacy). The community will also inspire the next generation of low-code app builders to learn and expand their skill sets with decentralized (web3) technologies. 
+
+# Current Code of Conduct
+[Farmworker Wallet OS Community Code of Conduct](https://github.com/Entidad/.github/blob/main/CODE_OF_CONDUCT.md)
+
+# TAC Sponsor
+_Include the name of a sponsor from the TAC, if identified (a sponsor helps mentor projects)_
+
+# Project License
+[Apache License, Version 2.0](https://www.apache.org/licenses/LICENSE-2.0)
+
+# Source Control
+Github
+
+# Issue Tracker
+Github
+
+# External Dependencies
```

Comment:
Are the CICD elements also supplied by GitHub?
And library/package hosting also supplied by GitHub?